keyboards/xd75/keymaps/arpinfidel/rules.mk

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013 Jun Wako <wakojun@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+ifndef QUANTUM_DIR
```

Comment:
If you could remove the ifdef and include lines here?  They're a hold over from the old makefile system and should be removed.

Basically you should have just:
```makefile
BOOTMAGIC_ENABLE = no
BACKLIGHT_ENABLE = yes
```